api/src/controllers/kots/KotsAPI.ts

```diff
@@ -227,6 +227,8 @@ export class KotsAPI {
     };
   }
 
+  // This handler was moved to kotsadm (go) in 1.14.0
+  // Remind me to delete this in 1.16.0
```

Comment:
why wait for 1.16 to delete it?

Reply:
I think there are some customers who have automation/integration tests that use this API directly, and it will give us a little time to help them move